server/src/test/java/org/spine3/server/reflect/Given.java

```diff
@@ -0,0 +1,111 @@
+ */
+
+package org.spine3.server.reflect;
+
+import com.google.common.util.concurrent.MoreExecutors;
+import org.spine3.base.EventContext;
+import org.spine3.server.event.EventBus;
+import org.spine3.server.event.EventStore;
+import org.spine3.server.storage.EventStorage;
+import org.spine3.server.storage.memory.InMemoryStorageFactory;
+import org.spine3.test.reflect.ProjectId;
+import org.spine3.test.reflect.command.CreateProject;
+import org.spine3.test.reflect.event.ProjectCreated;
+import org.spine3.test.reflect.event.ProjectStarted;
+import org.spine3.test.reflect.event.TaskAdded;
+
+import static org.spine3.base.Identifiers.newUuid;
+
+@SuppressWarnings("EmptyClass")
+/* package */ class Given {
+
+    /* package */ static class AggregateId {
+
+        private AggregateId() {
+        }
+
+        public static ProjectId newProjectId() {
+            final String uuid = newUuid();
+            return ProjectId.newBuilder().setId(uuid).build();
+        }
+
+    }
+
+    /* package */ static class EventMessage {
+
+        private static final ProjectId DUMMY_PROJECT_ID = AggregateId.newProjectId();
+        private static final ProjectCreated PROJECT_CREATED = projectCreatedMsg(DUMMY_PROJECT_ID);
+
+        private EventMessage() {
+        }
+
+        public static ProjectCreated projectCreatedMsg() {
+            return PROJECT_CREATED;
+        }
+
+        public static ProjectCreated projectCreatedMsg(ProjectId id) {
+            return ProjectCreated.newBuilder().setProjectId(id).build();
+        }
+
+    }
+
+    /* package */ static class Command{
+
+        private Command() {
+        }
+
+        /**
+         * Creates a new {@link org.spine3.test.reflect.command.CreateProject} command with the generated project ID.
+         */
+        public static CreateProject createProjectMsg() {
+            return CreateProject.newBuilder().setProjectId(AggregateId.newProjectId()).build();
+        }
+
+        /**
+         * Creates a new {@link CreateProject} command with the given project ID.
+         */
+        public static CreateProject createProjectMsg(ProjectId id) {
+            return CreateProject.newBuilder().setProjectId(id).build();
+        }
+
+    }
+
+    /* package */ static class Event{
+
+        private Event() {
+        }
+
+        /**
+         * Creates a new event bus with the {@link InMemoryStorageFactory}.
+         */
+        public static EventBus newEventBus() {
+            final EventStorage storage = InMemoryStorageFactory.getInstance().createEventStorage();
+            final EventStore store = EventStore.newBuilder()
+                                               .setStreamExecutor(MoreExecutors.directExecutor())
+                                               .setStorage(storage)
+                                               .build();
+            final EventBus eventBus = EventBus.newInstance(store);
+            return eventBus;
+        }
+
+    }
+
```

Comment:
Pls remove extra empty lines  at the bottom of classes. See, in *all* other classes there are no extra lines. And we need to be consistent.